platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformSchedulersConfig.java

```diff
@@ -137,5 +137,8 @@ public record PlatformSchedulersConfig(
         @ConfigProperty(defaultValue = "SEQUENTIAL CAPACITY(60) UNHANDLED_TASK_METRIC")
                 TaskSchedulerConfiguration stateGarbageCollector,
         @ConfigProperty(defaultValue = "200ms") Duration stateGarbageCollectorHeartbeatPeriod,
+        @ConfigProperty(defaultValue = "SEQUENTIAL CAPACITY(60) UNHANDLED_TASK_METRIC")
+                TaskSchedulerConfiguration signedStateSentinel,
```

Comment:
In this rare instance, I think the default capacity might be appropriate. Default capacity is 1. This is more than sufficient for the sentinel, since there is only the heartbeat going in, and that has offer semantics. You can simply remove `CAPACITY(60)`.

When we get around to refactoring this so that it doesn't use a static stateful thing to track states, we will probably need to reevaluate the capacity of this component's queue.

Reply:
removed